src/clr-angular/data/datagrid/providers/column-toggle-buttons.service.ts

```diff
@@ -7,28 +7,20 @@ import { Injectable, TemplateRef } from '@angular/core';
 import { Observable } from 'rxjs';
 import { Subject } from 'rxjs';
 
-export type ColumnToggleButtons = 'ok' | 'selectAll';
+export type ColumnToggleButtons = 'selectAll';
 
 @Injectable()
 export class ColumnToggleButtonsService {
   buttons: TemplateRef<any> = null;
   selectAllDisabled: boolean = false;
 
-  private _okButtonClicked = new Subject<void>();
-  public get okButtonClicked(): Observable<void> {
-    return this._okButtonClicked.asObservable();
-  }
-
   private _selectAllButtonClicked = new Subject<void>();
   public get selectAllButtonClicked(): Observable<void> {
     return this._selectAllButtonClicked.asObservable();
   }
 
   public buttonClicked(type: ColumnToggleButtons): void {
     switch (type.toLowerCase()) {
-      case 'ok':
```

Comment:
Waaaat?

A switch statement with only one case? Why?

Reply:
Same comment as above.

Reply:
Should be a basic `if (type.toLowerCasee() === 'selectall') { ... }`

Reply:
I think that is better. Will update.